src/policy/policy.h

```diff
@@ -46,6 +46,13 @@ static const unsigned int MAX_STANDARD_P2WSH_SCRIPT_SIZE = 3600;
  * only increase the dust limit after prior releases were already not creating
  * outputs below the new threshold */
 static const unsigned int DUST_RELAY_TX_FEE = 1000;
+/**
+ * Dogecoin: Default dust limit that is evaluated when considering whether a
+ * transaction output is required to pay additional fee for relay and inclusion
+ * in blocks. Overridden by -dustlimit
+ */
+static const unsigned int DEFAULT_DUST_LIMIT = 100000000;
```

Comment:
Can we express this relative to `COIN` please, I think `COIN / 100`? Makes it easier to read.

Reply:
Yeah that can help. For this change that adds the parameter, I don't want to do the default policy change yet, just add the functionality, so it'll be `COIN / 1` right now and we can in a subsequent policy PR add zeros instead of removing zeros. For consistency, I'll also change the `DUST_RELAY_TX_FEE` to be expressed in `COIN`?

Reply:
Sounds good!